pkg/cmd/controller/pipeline/pipelinerunner_controller.go

```diff
@@ -108,13 +110,19 @@ func (c *controller) startWorkers(ctx context.Context, wg *sync.WaitGroup, cance
 			Handler: mux,
 		}
 
+		logger.Debugf("about to spawn starting HTTP server on %s port %d", c.bindAddress, c.port)
```

Comment:
see above

Reply:
We *definitely* need more logging here, it's impossible to see what is going on otherwise. In particularly silently dropping errors is not good.

Reply:
I think here due to the nested go routines that some extra logging is very helpful to aid debugging.

Reply:
Some comments would also be a good idea ;-)

Reply:
Ok, removed that debugging, it's the silent dropping of errors that bother me